plugin/src/sbt-test/sbt-tpolecat/scalacOptions/build.sbt

```diff
@@ -185,6 +187,27 @@ val Scala31Options =
     "-source",
     "3.0-migration"
   )
+val Scala33Options =
+  Seq(
+    "-encoding",
+    "utf8",
+    "-deprecation",
+    "-feature",
+    "-unchecked",
+    "-language:experimental.macros",
+    "-language:higherKinds",
+    "-language:implicitConversions",
+    "-Ykind-projector",
+    "-Wvalue-discard",
+    "-Wunused:implicits",
+    "-Wunused:explicits",
+    "-Wunused:imports",
+    "-Wunused:locals",
+    "-Wunused:params",
+    "-Wunused:privates",
+    "-source",
+    "3.0-migration",
```

Comment:
sbt-typelevel was making this mistake as well. It seems this is not the sort of thing a plugin should be doing. But that will have to be fixed in another PR / breaking release.

> Without the project author explicitly opting into it. I think -source is dangerous and should not be enabled without thinking through all the consequences (and ideally it should only be used temporarily during a migration)

- https://github.com/lampepfl/dotty/discussions/17591#discussioncomment-6003475

Reply:
Don't worry - this isn't actually enabled by default, it's just enabled in the test suite to make sure that users can manipulate options for each mode separately.

Reply:
(see https://github.com/typelevel/sbt-tpolecat/pull/147/files#diff-25d6f7f6dbab5ef8591d0bce4e831ab4f799818e2d946938635f0f87c0a489afR25)